build.sbt

```diff
@@ -301,35 +489,90 @@ lazy val asyncHttpClient = libraryProject("async-http-client")
     startYear := Some(2016),
     libraryDependencies ++= Seq(
       Http4sPlugin.asyncHttpClient,
-      fs2ReactiveStreams,
+      fs2ReactiveStreams.value,
       nettyBuffer,
       nettyCodecHttp,
-      reactiveStreams,
+      reactiveStreams
     ),
     Test / parallelExecution := false
   )
   .dependsOn(core, testing % "test->test", client % "compile;test->test")
 
+lazy val domCore = libraryProject("dom-core", CrossType.Pure, List(JSPlatform))
+  .settings(
+    description := "Base library for dom http4s client and apps",
+    startYear := Some(2021),
+    libraryDependencies ++= Seq(
+      scalaJsDom.value.cross(CrossVersion.for3Use2_13),
```

Comment:
`for3Use2_13` usually results in pestilence and death downstream.  Anywhere it's previously been in the build, we've used it speculatively compile the module for Scala 3 so we're ready when the dependency is, but not inflict diamond dependencies on the users.  Unless it's somehow different in the scala.JS world, I think we need to limit this to Scala 2.

Reply:
200% agree. I PRed the Scala 3 cross-build into that repo and then this happened https://github.com/scala-js/scala-js-dom/issues/451. The proper release is on the way soon, if we can bear with.

Reply:
:+1: As long as we remember not to publish it for Scala 3 until that happens, this is great.